pkg/agent/util/net_linux.go

```diff
@@ -476,54 +462,52 @@ func RestoreHostInterfaceConfiguration(brName string, interfaceName string) {
 	klog.V(4).InfoS("Restoring bridge config to host interface")
 	bridgedName := GenerateUplinkInterfaceName(interfaceName)
 	// restore if interface eth0~ exists
```

Comment:
```suggestion
	// return if interface eth0~ doesn't exist
```

Reply:
Changed to "restore only when interface eth0~ exists"